client/src/components/utils/Loading/style.css

```
@@ -0,0 +1,12 @@
+.load-Com {
+    width: 100%;
+    height: 100vh;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-size: 1.5rem ;
+  }
```

leave a new line after each selector 
```suggestion
  }
  
```